backend/src/main/kotlin/app/ehrenamtskarte/backend/regions/database/Setup.kt

```diff
@@ -59,7 +62,7 @@ fun insertOrUpdateRegions() {
             }
         }
         // TODO #1551: Adjust regionidentifier_unique constraint
-        createOrUpdateRegion(NUERNBERG_PASS_PROJECT, "Nürnberg", "Stadt", null, "https://nuernberg.de")
-        createOrUpdateRegion(KOBLENZ_PASS_PROJECT, "Koblenz", "Stadt", "07111", "https://koblenz.de/")
+        createOrUpdateRegion(NUERNBERG_PASS_PROJECT, "Nürnberg", "Stadt", null, "https://nuernberg.de", false)
```

Comment:
:question: Wouldn't it make sense to read these values from the build config? Otherwise the config is a little spread between different places, isn't it?

Reply:
good point. But where should it be put? The project configs are in the react / administration part. 
We only could add it to the config.yaml.
@f1sh1918 Any preferences?

Reply:
Oh sorry, I missed that there are no build configs for the backend. Then nvm.

Reply:
Oh sorry, I missed that there are no build configs for the backend. Then nvm.

Reply:
i think the `config.yaml` is fine. yes it would be good to have one buildConfig for all project parts. The problem is that we use different source types in the backend which we may have to translate

Reply:
ok, what do you mean by "The problem is that we use different source types in the backend which we may have to translate". I dont understand this.

Reply:
this is just a side note that when we created a big `projectConfig` in f.e. typescript we can not simply import it for the backend but have to convert it to valid `yaml` syntax